contribs/commercialTrafficApplications/src/test/java/org/matsim/contrib/commercialTrafficApplications/jointDemand/commercialJob/DefaultCommercialJobGeneratorTest.java

```diff
@@ -24,7 +25,7 @@
     @Rule
     public MatsimTestUtils utils = new MatsimTestUtils();
 
-
+    //TODO: This test is not needed anymore! It gets not executed and DefaultCommercialJobGenerator requires injection
```

Comment:
from my side, please go ahead and remove the test :)

Reply:
Test removed